sysutils/salt/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem        1.0
 
 name               salt
-version            2017.7.3
+version            2017.7.4
 categories         sysutils python
 platforms          darwin
 maintainers        {@aphor gmail.com:jeremy.mcmillan}
```

Comment:
Would you be willing to tack "openmaintainer" on the end here? It allows other people to submit minor version updates and the like without you having to explicitly give permission.

Reply:
That sounds fine to me. Do you want to commit the change for openmaintainer, or do you want me to update this PR?

Reply:
It would be easiest if you update this PR. (Make sure to squash the commits.)